plugins/available/alias-completion.plugin.bash

```diff
@@ -63,14 +63,17 @@ function alias_completion {
             local compl_func="${new_completion/#* -F /}"; compl_func="${compl_func%% *}"
             # avoid recursive call loops by ignoring our own functions
             if [[ "${compl_func#_$namespace::}" == $compl_func ]]; then
+                if [[ $compl_func == _filedir_xspec ]]; then
```

Comment:
I think it makes sense to add a comment above this line to explain why this is needed, maybe with a link back to the issue #1497.

Reply:
Done